po/ru.po

```diff
@@ -5,18 +5,18 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PhotoCollage 1.4.3\n"
+"Project-Id-Version: PhotoCollage 1.4.4\n"
```

Comment:
No need to update `ru.po` in this PR, can you exclude it from this PR?

Reply:
Ok